app/message-pin/client/pinMessage.js

```diff
@@ -1,19 +1,25 @@
 import { Meteor } from 'meteor/meteor';
+import toastr from 'toastr';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
 
 import { settings } from '../../settings';
 import { ChatMessage, Subscriptions } from '../../models';
 
 Meteor.methods({
 	pinMessage(message) {
 		if (!Meteor.userId()) {
+			toastr.error(TAPi18n.__('error-pinning-message'));
 			return false;
 		}
 		if (!settings.get('Message_AllowPinning')) {
+			toastr.error(TAPi18n.__('error-pinning-message'));
```

Comment:
We shouldn't throw the same error for different motives, here we should say something like `pinning is not allowed`